doc/content/guides/getting-started/cli/_index.md

```diff
@@ -158,3 +184,9 @@ $ ttn-lw-cli end-devices create app1 dev2 \
 >Note: The `NwkSKey` is returned as `f_nwk_s_int_key` ({{% tts %}} uses LoRaWAN 1.1 terminology).
 
 >Hint: You can also pass `--with-session` to have a session generated.
+
+## Working With Data
```

Comment:
This does not belong in the CLI guide.

Reply:
Sorry, did a quick copy paste. This is also why Travis is not happy.